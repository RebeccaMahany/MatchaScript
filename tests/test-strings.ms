function void my_main() {
    string a = "I am a string but you won't see me";
    string b = "hey I am a string hello";
    a = b;
    print(a);
}

my_main();