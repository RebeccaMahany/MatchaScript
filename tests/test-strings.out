hey I am a string hello
